### DISCUSSION TITLE
Issue with useGLTF

### DISCUSSION THREAD

**Discussion by GDTNguyen** [pmndrs/drei#591] [Q&A] [answered]
Getting a Type error when using a min.glb file, TypeError: import object field 'a' is not an Object. 

`import { useGLTF, MeshDistortMaterial, Shadow } from '@react-three/drei'`
`const { nodes } = useGLTF(static/glb/geo.min.glb)`

Was working around 10 days ago, attention was brought to it recently. The react component using the glb file is wrapped in a react Suspense.

**Comment by drcmda** [comment, marked as answer]
this is not a path string -> useGLTF(static/glb/geo.min.glb)

but in any case, update drei, which now uses a versioned draco cdn service. googles draco compression cdn service went down, every project using draco suffered from it.

**Reply by GDTNguyen** [reply]
thanks a bunch! I updated my packages to lastest versions.
